shift/notebooks/adult_covariate_shift.md

```diff
@@ -0,0 +1,141 @@
+# AutoGluon.Shift Tutorial
+
+## Introduction
+
+Distributional shift is when there is a difference between the training and test data in a prediction problem.  In this tutorial we introduce the `C2STShiftDetector` class which will detect and explain a change in the covariate (X) distributions, a phenomenon that we call XShift.  This is one of the ways in which distributional shift can manifest, but not the only one.
+
+
+```python
```

Comment:
We can change all ` ```python ` to ` ```{.python .input} `, and move the file into https://github.com/awslabs/autogluon/tree/master/docs/tutorials . We may create a new "shift" folder under the tutorial folder.

Reply:
This is done